scripts/arm32_ci_script.sh

```diff
@@ -269,6 +264,12 @@ do
     esac
 done
 
+if [[ $__linuxCodeName == "xenial" || $__linuxCodeName == "tizen" ]]; then
+    # This case does not support CI build yet.
```

Comment:
Because I'm not sure the 'latest-or-auto-docker' image works well for Ubuntu16.04 arm CI. So I wanna make a dummy PR to confirm the operation.